File: YAzyki programmirovaniya/5 zadach' po fizike na 7 yazykakh programmirovaniya/Задачи VB/WindowsApplication1/Form2.vb
Public Class Form2
    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        Form1.Show()

    End Sub

    Private Sub button4_Click(sender As Object, e As EventArgs) Handles button4.Click
        Form5.Show()

    End Sub

    Private Sub button3_Click(sender As Object, e As EventArgs) Handles button3.Click
        Form4.Show()

    End Sub

    Private Sub button2_Click(sender As Object, e As EventArgs) Handles button2.Click
        Form3.Show()

    End Sub

    Private Sub button5_Click(sender As Object, e As EventArgs) Handles button5.Click
        Form6.Show()

    End Sub

    Private Sub menuStrip1_ItemClicked(sender As Object, e As ToolStripItemClickedEventArgs)

    End Sub

    Private Sub выходToolStripMenuItem_Click(sender As Object, e As EventArgs)
        Close()

    End Sub

    Private Sub Form2_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub
End Class

File: programming languages/5_physics_problems_in_7_programming_languages/Задачи C#/Задачи/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Задачи
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }

        private void button3_Click(object sender, EventArgs e)
        {
            Form4 Form4 = new Form4();
            Form4.ShowDialog();
        }

        private void выходToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            Form1 Form2 = new Form1();
            Form2.ShowDialog();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            Form3 Form3 = new Form3();
            Form3.ShowDialog();
        }

        private void button4_Click(object sender, EventArgs e)
        {
            Form5 Form5 = new Form5();
            Form5.ShowDialog();
        }

        private void button5_Click(object sender, EventArgs e)
        {
            Form6 Form6 = new Form6();
            Form6.ShowDialog();
        }

        private void Form2_Load(object sender, EventArgs e)
        {

        }
    }
}
